File: Extracting Numbers.py
class Solution:
    # @param A : string
     # @return an long
    def solve(self, A):
        d=''
        su=0
        for i in range(len(A)):
            #print(A[i],d)
            if A[i].isdigit():
                d+=A[i]
            else:
                if d!='':
                    su+=int(d)
                    d=''
        if d!='':
            su+=int(d)
        return su
            

File: Level 1/Time Complexity/Extracting Numbers.java
public class Solution {
    public Long solve(String A) {
        String temp="0";
        long sum=0;
        for(int i=0;i<A.length();i++){
            if(Character.isDigit(A.charAt(i))){
                temp+=A.charAt(i);
                if(i==(A.length()-1)){
                  sum=sum+Long.parseLong(temp);  
                }
            }
            else{
                sum=sum+Long.parseLong(temp);
                temp="0";
            }
        }
        return sum; 
    }
}
